desktop-app-legacy/app/components/LiveCssEditor/index.js

```diff
@@ -1,6 +1,6 @@
-import React, {useMemo, useState, useEffect, useRef} from 'react';
-import {Rnd} from 'react-rnd';
-import {useSelector, useDispatch} from 'react-redux';
+import React, { useMemo, useState, useEffect, useRef } from 'react';
```

Comment:
what's the purpose of these changes in the legacy app?

Reply:
This would give an simplified version

Reply:
Better understanding

Reply:
@minowau We don't actively do any development on the legacy app. So, for the sake of clean git history, can you please revert these changes so that we can merge the PR?